index.js

```diff
@@ -232,4 +232,35 @@ form.addEventListener('submit', (event) => {
     form.submit();
   }
 });
+const formData = JSON.parse(localStorage.getItem('info')) || {
+  name: '',
+  mail: '',
+  msg: '',
+};
+const nameData = document.getElementById('name');
+const message = document.getElementById('msg');
+function saveToLocal(formData) {
+  localStorage.setItem('info', JSON.stringify(formData));
+}
+
+nameData.addEventListener('input', (event) => {
+  formData.name = event.target.value;
+  saveToLocal(formData);
+});
+
+email.addEventListener('input', (event) => {
+  formData.mail = event.target.value;
+  saveToLocal(formData);
+});
+
+message.addEventListener('input', (event) => {
+  formData.msg = event.target.value;
+  saveToLocal(formData);
+});
+function inputData() {
+  nameData.value = formData.name;
+  email.value = formData.mail;
+  message.value = formData.msg;
+}
+window.onload = inputData();
 console.log(closeP4(), closeP3(), closeP2(), closeP1());
```

Comment:
:red_circle: According to this [javascript best practices list](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) You should avoid committing ` console.log` to your repo. It’s ugly, it kills performance and it can make confidential data visible to anyone using the browser tools to look at your website. Kindly remove this line